uint8 index
float32 torque
float32 angle
float32 velocity
float32 kp
float32 kd
